contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -21,21 +17,23 @@ import { Errors } from "../../../lib/Errors.sol";
 /// @title Ip Royalty Vault
 /// @notice Defines the logic for claiming revenue tokens for a given IP
 /// @dev [CAUTION]
-///      Do not transfer ERC20 tokens directly to the ip royalty vault as they can be lost if the pendingVaultAmount
+///      Do not transfer ERC20 tokens directly to the ip royalty vault as they can be lost if the poolInfo
 ///      is not updated along with an ERC20 transfer.
-///      Use appropriate callpaths that can update the pendingVaultAmount when an ERC20 transfer to the vault is made.
-contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, ReentrancyGuardUpgradeable {
+///      Use appropriate callpaths that can update the poolInfo when an ERC20 transfer to the vault is made.
+contract IpRoyaltyVault is IIpRoyaltyVault, ERC20Upgradeable, ReentrancyGuardUpgradeable {
     using EnumerableSet for EnumerableSet.AddressSet;
-    using SafeERC20Upgradeable for IERC20Upgradeable;
+    using SafeERC20 for IERC20;
 
     /// @dev Storage structure for the IpRoyaltyVault
     /// @param ipId The ip id to whom this royalty vault belongs to
-    /// @param lastSnapshotTimestamp The last snapshotted timestamp
-    /// @param pendingVaultAmount Amount of revenue token pending to be snapshotted
-    /// @param claimVaultAmount Amount of revenue token in the claim vault
-    /// @param claimableAtSnapshot Amount of revenue token claimable at a given snapshot
-    /// @param isClaimedAtSnapshot Indicates whether the claimer has claimed the revenue tokens at a given snapshot
+    /// @param lastSnapshotTimestamp [DEPRECATED] The last snapshotted timestamp
+    /// @param pendingVaultAmount [DEPRECATED] Amount of revenue token pending to be snapshotted
+    /// @param claimVaultAmount [DEPRECATED] Amount of revenue token in the claim vault
+    /// @param claimableAtSnapshot [DEPRECATED] Amount of revenue token claimable at a given snapshot
+    /// @param isClaimedAtSnapshot [DEPRECATED] Indicates whether the claimer has claimed the token at a given snapshot
     /// @param tokens The list of revenue tokens in the vault
+    /// @param poolInfo The accumulated balance of revenue tokens in the vault
```

Comment:
Should we rename the two new variables away from "Info"?

For example
`vaultInfo` rename to `vaultAccBalance`
`claimerInfo` rename to `revenueTokenDebt`

Reply:
renamed.